config/routes.rb

```diff
@@ -2,5 +2,7 @@
   # Rails.application.routes.draw do
   resources :users, only: %i[index show] do
     resources :posts, only: %i[index show]
+    resources :comments, only: [:create]
+    resources :likes, only: [:create]
```

Comment:
- [ ] Good job so far but unfortunately I can't find  GET method  `posts#new ` to get a post and POST method `posts#create ` to create a new post and also `comments#new`. Please fix it to make your app works as expected :+1: